src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingDetails.cs

```diff
@@ -99,12 +99,11 @@ public void VerifyViewModelIsMappedFromRequest()
                 Assert.AreEqual(expectedTotalCost, _viewModel.TotalCost, "The total cost stored in the model is incorrect");
             }
 
-            public void VerifyResultIsRedirectToV1()
+            public void VerifyResultIsNotRedirectToV1()
```

Comment:
As we're removing the redirect altogether, I suggest we delete this test completely, so that we're not left with tests around things we're _not_ doing.